hack/vendor/olm-test-script/e2e-olm.sh

```diff
@@ -59,9 +59,9 @@ test::object_assert() {
 }
 
 # Name of the configmap that we will create
-CONFIGMAP_NAME=${CONFIGMAP_NAME:-openshift-olm-test}
+CONFIGMAP_NAME=${CONFIGMAP_NAME:-openshift-olm-test$SUFFIX}
 
-CRD=$(sed '/^#!.*$/d' $MANIFEST_DIR/$VERSION/*crd.yaml | grep -v -- "---" | indent apiVersion)
+CRD=$(cat $MANIFEST_DIR/$VERSION/*crd.yaml | sed '/^#!.*$/d' | grep -v -- "---" | indent apiVersion)
```

Comment:
what was wrong with the prior implementation?

Reply:
> what was wrong with the prior implementation?

It didnt properly indent and format multiple crd files for the configmap.